### DISCUSSION TITLE
"Big Bang: Tweak it or throw it out? "

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#39] [Cosmology Models]
Eric Lerner on YouTube [https://youtu.be/FhqqJGYw9Fw](https://youtu.be/FhqqJGYw9Fw)

*In the sixth episode of our video series, “JWST and the Big Bang Never Happened Debate”, LPPFusion Chief Scientist Eric J. Lerner replies to those who say “there are just a few things wrong with the Big Bang — just needs tweaking”. In fact, there are far more wrong predictions that right ones for the Big Bang theory, something that is widely known in the field, but not widely acknowledged...*

**Comment by khuramonline** [comment]
Even more fundamental problems ... Maybe ACG also might not acknowledge.

When Expanding Universe has so many problems then it is better to assume a non-expanding universe.

When it is non-expanding universe then first need is to get rid of ramnants of all the Expanding Universe based methods, formulas or expectations.

There can be ACG members who, after acknowledging that universe is not expanding, may continue to expect "bang" or even "bangs".

Alternative group may discard the idea of Expanding Universe but distance of any galaxy may continued to be determined by the expanding universe based formulas ... 

A galaxy with redshift 13 may continued to be acknowledged to be located at distance of 13.5 billion light years.

The need to remove the formula z=v/c or any further derivation of this formula may not be recognized even by the alternative group.

Alternative group may continue to think that after 13.6 billion light years there is some real "dark ages" (which is expanding universe based idea) and anything visible within or beyond such distance may be continued to be regarded as "noise".

At least it is better stance than that of official science mainstream group. Because they do not even call it noise. They only pretend that anything or any noise or whatever does not even exist. They do not talk about any such thing using any term. They just pretend to be blind about those things which an alternative group may acknowledge as "noise".

**Reply by OlliSant** [reply]
Of course there are bangs. Every star is from some cloud, and every cloud is from some explosion or crunch. That is the dynamic of the universe, but explosions are local. No beginning, no expansion for the whole universe. We can suppose that, and then make a theory that is better. If it  is not  better then it is not better, but must try.

**Comment by HanDeBruijn** [comment]
It's not that simple. There exist cosmological theories where in one (time) reference frame there is a beginning (say "bang") and in another reference frame the universe is eternal, without bangs. And by the way, why can't a beginning start __quietly__?
More in general, I think that the glorification of __violence__ is bankrupting physics and cosmology. Think about particle accelerators for example, especially the [Large Hadron Collider](https://en.wikipedia.org/wiki/Particle_Fever). As if Depth of Truth is proportional to Heaviness of Bombardment. As if all this smashing of matter can reveal anything else than space debris.
It is clear, however, that modern science is herewith just following a common trend in our nowadays "civilized" society, where you can hardly watch a movie without violent bangs in it :-(

**Reply by OlliSant** [reply]
The solution is so simple. The universe is eternal, but the local part- universes begin. Think of this thoroughly. 

The local is those galaxies that are from the same beginning as our Milky Way. And there is other local universes too. It does not matter if the beginnings are bangs or something else. A smoke anyway.

For our brains it is difficult to understand "all", "everything", "everywhere", "always", but in the universe there are such things, and they are not absolutes, "all" makes them not absolutes.

The universe, the whole universe is eternal, and the space of the whole universe is static. But our local universe is dynamic. Bangs and smokes and forming of stars, and bangs again. Eternally, locally.

So simple, to make difference between the whole and the parts, when you understand this, but so impossible to understand for people and cosmologists and everybody. In science we must make a theory and prove it, otherwise all this thinking is of no use. I admit that.

**Comment by khuramonline** [comment]
In year 2017, I had preliminary ideas about local universes in the form of various sectors of the total universe. I so far consider these ideas not mature enough and I did not include these preliminary things in my books (2018) and (2019).

But If I get time in life to actually sort out these matters in details then I will review and improve these preliminary ideas. I am sharing here but with the provision that I treat it as a preliminary and not a finalized stance:

[Are there multiple universes?](https://www.quora.com/Are-there-multiple-universes/answer/Khuram-Rafique)

Second issue is about local explosions. Yes Supernovas are reality but it is not universal reality. Not all stars go through this stage or process and number of actually observed supernovas are quite limited. Here again, I have a preliminary thing in the form of a question. To me, for a non-expanding and much older universe than just 14 billion years, the right answer to following question is YES i.e. heavior elements can be "cooked" over long time as well. So following is the question that I asked on quora:

[If supernova is 10 billion times more energetic than the base star then why are heavier elements produced only through supernova and not due to continuous exposure of star's own energy over the period of 10 billion years?](https://www.quora.com/If-supernova-is-10-billion-times-more-energetic-than-the-base-star-then-why-are-heavier-elements-produced-only-through-supernova-and-not-due-to-continuous-exposure-of-stars-own-energy-over-the-period-of-10-billion)

**Reply by OlliSant** [reply]
Explosions must be much bigger than supernovas, if there are local beginnings.

**Reply by khuramonline** [reply]
If initially there were only scattered particles that prevailed everywhere then it is not the case of explosions. In such a case, more likely is the case of condensation. And for the initial phase, that was very slow condensation.

Rather than an "expanding universe", it is actually a "condensing universe".

For example today's clusters are condensing to form bigger galaxies. In earlier times, smaller galaxies were formed by condensing the smaller clusters of that time.

My estimate is that in the current universe that we see, 4-5 or even more major cluster mergers already have taken place at observable universe scale. Future galaxies are going to be even larger with greater in-between distances. That is what I mean by "condensing universe".

**Reply by OlliSant** [reply]
Rationally best philosophical theory is eternal and boundless universe. Space is immensely big, boundless, and has no outside, everything is there. Matter is galaxies and forces. Galaxies move as they move, and are in equilibrium. Time is eternal. No more space coming, no more matter. Time is what is coming more, and everything is relative. 

The whole universe, the observable universe and the local part- universe are all existing and are different things, not the same thing.  The whole universe is much bigger than the observable universe. The theory of the whole universe is different as the theory of the local universe. Both are made by studying the observable universe and by making these two theories. BB is rationally wrong and impossible, not a theory of the whole universe as it should be.

So, philosophy is most important and physics comes next and mathematics is only a tool, and language in physics, not in philosophy.

**Reply by ftessier** [reply]
@OlliSant Are space and time _fundamental_ — philosophically? Are they axiomatic in this ontology?

**Reply by OlliSant** [reply]
Space and time (movement) and matter are the concepts (and objective things in the reality) what we need. Or we must invent better concepts for the things in the reality. And they are relative to each others in the universe. But in the case of the universe everything is different as in every other things. People always forget that universe is universe. It is somehow eternal and infinite, otherwise it is not universe but a smaller and different thing. Eternity might be fundamental, absolute, but space definitely is not, and time is what is coming more, not space, not matter. In the case of the universe. Part- universes, parallel part- universes and observable universe are different things. Only one whole, total, eternal universe.

**Reply by ftessier** [reply]
Our conception of space and time, it seems to me, are derived from a uniquely human perspective; I mean specific to the scale of our detectors, brains included. In the past, large shifts in philosophy have been correlated to escaping the narrow human experience: Thales, Anaximander, Aristotle, Copernicus, Galileo, Darwin, Einstein, Feynman, to name the usual suspects. I trust that disentangling space and time from our human view will be no exception and allow a richer way to contemplate the world. I consider it precarious to found philosophies on human-specific perceptions.

**Comment by HanDeBruijn** [comment]
I wrote: _It's not that simple. There exist cosmological theories where in one (time) reference frame there is a beginning (say "bang") and in another reference frame the universe is eternal, without bangs. And by the way, why can't a beginning start quietly?_

With the [variable mass theory](https://www.youtube.com/watch?v=T7jVviVBzKs) we have two clocks, an Orbital clock and an Atomic clock. It's a matter of taste which one of the two you are willing to accept as the "true" timekeeper. Other names for the two timeframes are "Kinematic/Dynamic time" (: Arthur Milne) and "Einstein/Minkowski frame" (: Fred Hoyle).
Now let $H=$ Hubble parameter, $T=$ orbital time, $t=$ atomic time, $T_0=t_0$ <i>synchronization</i> timestamp of the two clocks. Then the following formula is valid, as is proved in [A Basic Equation](http://www.alternatievewiskunde.nl/cosmology/simpel.htm#opnieuw).

$$
\left[1+\frac{1}{2}H(T-T_0)\right]^2 = \Large e^{H(t-t_0)}
$$

In Atomic time (right hand side) there is no beginning: the universe is eternal. In Orbital time (left hand side) there <b>is</b> a beginning !! And that beginning is smooth. Let's call this "bang" $A$ . Our smooth start is found by putting the left hand side to zero for $T=A$ . Then we easily find that $T_0-A=2/H$, meaning that the current "age of the universe" is $2/H$, not $1/H$ as according to many other models.

OlliSant writes: _philosophy is better than physics to study the universe._
HanDeBruijn says: _mathematics is better than philosophy to study the universe._

**Reply by RedshiftDrift** [reply]
> OlliSant writes: *philosophy is better than physics to study the universe.*
> HanDeBruijn says: *mathematics is better than philosophy to study the universe.*

@marmetl knows: *physics is better than mathematics to study the universe.*

... "is better than" is a non transitive relationship 😄

**Reply by budrap00** [reply]
We're on the same page there Louis!

**Reply by HanDeBruijn** [reply]
Galileo Galilei: _The laws of nature are written by the hand of God in the language of mathematics._

**Reply by RedshiftDrift** [reply]
*God* and *mathematics* are human inventions designed to deal with the real and the imaginary.
They both offer counsel, but they are not the only available tools.
We *__read__ the laws of nature as if they were written by the hand of God in the language of mathematics.*
But it's just a language, not nature itself.

Only *physics* is specifically designed to study nature, and this can be done *without god* and [*without mathematics*](https://cosmology.info/essays/criteria-for-a-scientific-model_marmet.html).

**Reply by RichardRavenhall** [reply]
The above states absolutes and I do not believe in absolutes. That only physics is specifically designed to study nature is contested.  When observations and statistics are available, we can get into the ballpark in a fraction of the time and from there get some idea of the physics. This is the ongoing debate between engineers and physicists, and we are never going to find common ground on it.

**Reply by RedshiftDrift** [reply]
Do you contest the scientific method?  (It's an absolute!)

**Reply by HanDeBruijn** [reply]
Two concerns about your first sentence:
1. God has been _discovered_ by humans, not invented.
2. Mathematics has been _discovered_ by humans, not invented.

Originally so. You are right in that both concepts have been distorted later on. But then our task must be [_to get rid of those distortions_](http://www.alternatievewiskunde.nl/QED/klassiek.htm). Which is not going to be accomplished by throwing away the child with the bathwater.

About your [Criteria for a Scientific Model](https://cosmology.info/essays/criteria-for-a-scientific-model_marmet.html) article.
I find that it is incomplete. We as humans do not only _observe_ nature, we also _modify_ it, namely by our __technology__.
As is confirmed by quantum mechanics, there is no sharp distinction between observation and disturbance / our __making__ of reality.
As RichardRavenhall writes: _This is the ongoing debate_ [read: different practice] _between engineers and physicists_.
So the subtitle of your article could be augmented as: _The measuring device and human technology as physical concepts_.
Classical (undistorted) thermodynamics is an exellent example of how to build a good physics theory with technological concepts. Read about the [Carnot cycle](https://en.wikipedia.org/wiki/Carnot_cycle) (French physicist!) and the famous efficiency formula of [Carnot's theorem](https://en.wikipedia.org/wiki/Carnot%27s_theorem_(thermodynamics)). Quote: _the Carnot engine should be thought as the theoretical limit of macroscopic scale heat engines rather than a practical device that could ever be built._
This has been a guideline for my own theoretical work: not to find an exact description of the phenomenon (which is virtually impossible) but instead a useful _theoretical limit_. It is often sufficient for an engineer to know that _reality cannot be worse than theory_ (as quoted from [Ideal Internal Flow](http://www.alternatievewiskunde.nl/jaar2017/ideal.htm), about thermal stress in an apparatus). There are more examples of this approach on my website.

**Reply by RichardRavenhall** [reply]
Per Wikipedia, the scientific method is an empirical method for acquiring knowledge.  I agree with this definition and have no problem with its application. My issue relates to the goal and how to get there.  We are looking for some idea of what's really going on and can advance towards this without full knowledge of it.  Why not go part way first and then use this information to look for more knowledge later?  Why make it a problem with 4 or 6 unknowns when it can be started with an approximate solution of only 2 unknowns?

**Reply by HanDeBruijn** [reply]
_Why make it a problem with 4 or 6 unknowns when it can be started with an approximate solution of only 2 unknowns?_

Agreed! Let's start with one of the main problems in cosmology, with only __one__ "unknown" involved. The picture is from a video by Eric Lerner at exactly one hour (1:00:00) from the start: [The Big Bang Never Happened--The Science of the Censored Papers](https://www.youtube.com/watch?v=ZlFpq49Ri8Y).

![lerner_half](https://github.com/a-cosmology-group/acg/assets/135433222/c9fe6a7c-6f1f-477c-9ec9-7c246db4d8f1)
Let $H=$ Hubble parameter, $G=$ Gravitational constant, $\rho_0=$ (mean) mass density here and now (at time and place of observer), then we have a couple of proposals. The first one is (theoretically) valid or not in our Atomic (time)frame:

$$
H = \sqrt{\chi G \rho_0} \quad \mbox{with} \quad \chi = 8\pi \times
\begin{cases}
  1/8 & (: \mbox{OI}) \\
  1/6 & (: \mbox{Mach}) \\
  1/3 & (: \Lambda\mbox{CDM}) \\
  0.3883945571 & (: \mbox{UAC})
\end{cases}
$$

Where [OI](http://www.alternatievewiskunde.nl/cosmology/Ghosh.htm), [Mach](http://www.alternatievewiskunde.nl/cosmology/Ghosh.htm), [&Lambda;CDM](http://www.alternatievewiskunde.nl/cosmology/numbers.htm), [UAC](http://www.alternatievewiskunde.nl/cosmology/helaas.htm) denote different cosmological models, a very much limited set studied by me.

Two other possibilities come with our Orbital (time)frame. Let $T_0=$ <i>synchronization</i> timestamp, $A=$ timestamp of our (smooth / violent) beginning. Then we have:

$$
H =
\begin{cases}
 1/(T_0-A) & (: \Lambda\mbox{CDM}) \\
 2/(T_0-A) & (: \mbox{UAC})
\end{cases}
$$

Galileo Galilei said: _The laws of nature are written by the hand of God in the language of mathematics._
Does someone notice why this statement is contradictory to my writeups? (Hint: _emphasis_ on technology)

No response. Okay, last hint. The _synchronization_ $t_0=T_0$ of the two clocks, by the hand of _who_ it is done?

**Reply by RedshiftDrift** [reply]
@HanDeBruijn 
> I find that it {[Criteria for a Scientific Model](https://cosmology.info/essays/criteria-for-a-scientific-model_marmet.html) article} is incomplete.

It was not meant to be complete $-$ completeness is not really possible as our metaphysics evolves with time.

> We as humans do not only observe nature, we also modify it, namely by our technology.

Our technology does not modify the laws of nature.

**Reply by RedshiftDrift** [reply]
@HanDeBruijn In addition to your proposals

$$
H = \sqrt{\chi G \rho_0} \quad \mbox{with} \quad \chi = 8\pi \times
\begin{cases}
  1/8 & (: \mbox{OI}) \\
  1/6 & (: \mbox{Mach}) \\
  1/3 & (: \Lambda\mbox{CDM}) \\
  0.3883945571 & (: \mbox{UAC})
\end{cases}
$$

and

$$
H =
\begin{cases}
 1/(T_0-A) & (: \Lambda\mbox{CDM}) \\
 2/(T_0-A) & (: \mbox{UAC})
\end{cases}
$$

there is also, based on a [simplified calculation](https://adsabs.harvard.edu/full/2009ASPC..413..268M) using the quantum theory of ponderomotive scattering,

$$ H_0 = \zeta \ \frac{\alpha h^2}{m_e^2 c} \ n_e  \mbox{\hspace{10pt} (: STz)}$$

expressed as a function of fundamental constants involved in electromagnetic interactions with electrons, where $\zeta \approx 0.6$ is a geometrical factor describing the light distribution that produces the Stimulated Transfer Redshift (STz), and $n_e \approx 0.3 \mbox{ electron/m}^3$ is the average number-density of free electrons in the observation path.

(Of interest to @sahil5d: comparing this equation with the general-relativistic versions given above, one gets a relationship between atomic constants and the gravitational constant that looks like the [*Large Numbers Coincidence*](https://medium.com/@sahil50/a-large-numbers-coincidence-299cb199e03d), but you need to rewrite it by including the fine structure constant $\alpha$ which is often explicit in a quantum theory of light like *photon Hubbling*.)

**Reply by HanDeBruijn** [reply]
@marmetl @sahil5d @PatrGT . ~~I know that Patrick G. Tonin has subscribed too but can't find his nickname here.~~
Anyway. Let LHS be the left hand side and RHS be the right hand side in Sahil's Large Numbers Coincidence formula, then:

$$
\large \frac{1}{\mbox{LHS}} = \frac{1}{\mbox{RHS}} \hspace{1pt} \mbox{(Gupta)} = \frac{\alpha_G}{2} \hspace{1pt} \mbox{(Tonin)}
$$

Here $\alpha_G$ is a dimensionless number in Patrick's theory. It can be (re)written __as desired__:

 $\alpha_G=$ ( [Newton's law](https://en.wikipedia.org/wiki/Newton%27s_law_of_universal_gravitation") )  / ( [Coulomb's law](https://en.wikipedia.org/wiki/Coulomb%27s_law) ) $\times$ ( [__Fine-structure constant__](https://en.wikipedia.org/wiki/Fine-structure_constant) )

Details as usual at my website: [Physical dimensions](http://www.alternatievewiskunde.nl/cosmology/numbers.htm#PT) .

**Reply by RedshiftDrift** [reply]
@PatrGT is Patrick, this mention should reach him...

**Reply by HanDeBruijn** [reply]
@marmetl : Couldn't resist to check dimensions and calculate the value of $H_0$ with your formula.

$$
H_0 = \zeta \ \frac{\alpha h^2}{m_e^2 c} \ n_e  \mbox{\hspace{10pt} (: STz)}
$$

Check on physical dimensions correctness (with $1=$ dimensionless) is OK:

$$
1/time = 1 \times \frac{1 \times (length^2\times mass/time)^2}{mass^2 \times length/time} \times 1/length^3 
$$
```
alpha := 1/137.035999084;
h := 6.62607015*10^(-34);
c := 299792458;
m_e := 9.1093837*10^(-31);
zeta := 0.6;
n_e := 0.3;
H := zeta*(alpha*h^2)/(m_e^2*c)*n_e;
Mpc := 3.08567758*10^22;
H := H*Mpc/1000;
```
And the outcome is $H_0=71.\enclose{horizontalstrike}{53222672}\hspace{1pt} (km/s)/Mpc$ , which is in close agreement with most other values for the Hubble.
But I kow that the formulas as presented by __me__ are often used he other way around, namely to get an estimate for the age of the universe and for the mean mass density $\rho_0$ around us.
So I'm wondering if the same is not the case for __your__ formula as well: do you have _independent sources_ for the geometrical factor $\zeta$ and for the average number-density $n_e$ of free electrons in the observation path?

**Reply by RedshiftDrift** [reply]
@HanDeBruijn *Dimensional analysis* is the name of what you do when you "check dimensions".  Scientists learn that in kindergarden, so there is no need to explicitly show how the calculation is done.  I have been doing this for 52 years and every scientist does it.

To the physicist, equations have a physical meaning [dimension].  The equation above

$$
H_0 = \zeta \frac{\alpha h^2}{m_e^2 c} n_e \quad \mbox{is the product of} \quad 
\begin{cases}
  \delta_z = \zeta \frac{h\nu}{mc^2} & \mbox{the redshift per scattering events } [1], \\
  \sigma = \frac{\alpha h}{m_e \nu} & \mbox{the scattering cross section } [area], \\
  n_e & \mbox{the electron number-density } [1/length^3], \\
  c & \mbox{the speed of light } [length/time],
\end{cases}
$$

which gives inverse $[time]$ as the dimension of the Hubble constant.

When an article presents equations with incoherent units, it means that the author did not understand the physical meaning of the equation.  Unless it's an obvious typo, the article is probably meaningless.

Scientists also learn in kindergarden that the displayed digits of a number also have a physical meaning.  The digits represent knowledge.  In maths, in makes sense to write $\pi = 3.1415926535897932384626433832795028841971...$ (I can't remember the rest :smile:)

However in the physical sciences, the most accurate measurements rarely have an absolute accuracy below $1\times 10^{-6}$, and in cosmology numbers are presented on a $\log_{10}$-scale because most astronomical measurements give "order-of-magnitude" results.  When engineers and mathematicians copy the output of their calculators to convey the result of their calculations 😮 it is a mistake that is just as bad as using incoherent [dimensions].  This shows that the author doesn't understand the physical meaning of the numbers they give.

That being said, I gave the values of $\zeta$ and $n_e$ to ***one*** significant digit because they have no physical meaning below 10% uncertainty.  The result $H_0 = 71.53222672$ km/s/Mpc is meaningless since even the best measurements of the Hubble constant by Riess have >1% uncertainties!

**Reply by RedshiftDrift** [reply]
@HanDeBruijn The geometrical factor $\zeta$ is obtained from:

$$
\zeta = \frac{\int_0^\pi R(\Theta) \ \sin^2 \left( \frac{\theta}{2} \right) \ \rm d \Theta}{\int_0^\pi R(\Theta) \ \rm d \Theta} \approx 0.6174...
$$

where

$$
R(\Theta) = \sin \frac{\Theta}{2} \ \sin \Theta \ \left[ 1 - \left( 1-\frac{2}{\pi} \right) \sin^2 \Theta \right]
$$

Each term has a physical meaning (light interference, intensity gradient, momentum recoil, etc.)  However, this equation is an approximation for isotropic and polarized light.  A more precise calculation would have to consider that between galaxies, light is not perfectly isotropic and is usually not polarized (the $1-2/\pi$ term is the result of another integral over polarization states.)

So although the value of $\zeta$ is mathematically defined exactly by the model, it depends on conditions in space that are not well known.  When the actual conditions found in space are considered in this calculation, the value should be near $\zeta = 0.6$.


The number-density of free electrons in the observation path is a measured quantity.  The integrated value $\int n_e(r) dr$ along the path from *source* to *us* is obtained from Fast Radio Bursts (FRBs) dispersion.  Again, a constant $n_e$ is an approximation since the density varies as a function of position in space.  Variations by a factor of two can be seen from one FRB to another.  I analyzed 12 FRBs a few years ago using the Euclidean metric of a static cosmology and obtained an average $n_e = 0.25_{-0.1}^{+0.2}$, a value comparable to the published values obtained in the ΛCDM framework.  There are now about 20(?) FRBs with a known redshift, but a new analysis would not reduce the uncertainty significantly.

To answer your question, the STz model has no adjustable parameter but depends on measurements of the light distribution along the observation path for $\zeta$ and the electron number-density $n_e$.  It predicts a value for the Hubble constant $\log_{10}(h) = 1.8\pm 0.3$ consistent with observations (the dimensionless Hubble constant $h$ is defined from $H_0 = h \times 100 km/s/Mpc$).

**Comment by OlliSant** [comment]
Scientific method is making theories and testing them. Mathematics is a tool in exact sciences but not in philosophy. And cosmology is always philosophy and astronomy, both. It was a big mistake to leave it to physicists.
